src/test/java/pl/solvd/carina/CarinaDemoHeaderTest.java

```diff
@@ -0,0 +1,125 @@
+package pl.solvd.carina;
+
+import com.qaprosoft.carina.core.foundation.IAbstractTest;
+import com.zebrunner.agent.core.annotation.TestLabel;
+import com.zebrunner.carina.core.registrar.ownership.MethodOwner;
+import com.zebrunner.carina.core.registrar.tag.Priority;
+import com.zebrunner.carina.core.registrar.tag.TestPriority;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class CarinaDemoHeaderTest implements IAbstractTest {
+    @Test()
+    @MethodOwner(owner = "ashvedava")
+    @TestPriority(Priority.P3)
+    @TestLabel(name = "feature", value = {"web", "regression"})
+    public void testZebrunnerLogoIsClickable() {
+        OverviewPage overviewPage = new OverviewPage(getDriver());
+        overviewPage.open();
+        Assert.assertTrue(overviewPage.isPageOpened(), "Overview page is not opened");
+        Header header = overviewPage.getHeader();
+        Assert.assertTrue(header.isUIObjectPresent(2), "Header element wasn't found!");
```

Comment:
it's parametr timeout. why the ability to use the timeout parameter if it cannot be used because of the unambiguous understanding of looking at the name ..?